docs/user_guides/fs/feature_view/transformation-function.md

```diff
@@ -1,75 +1,151 @@
+Creation of a Many to One transformation function is similar to that of One to One transformation function, the only difference being that the transformation function accepts multiple features as input.
 
 === "Python"
+    !!! example "Creation of a Many to One Custom Transformation Function in Hopsworks."
+        ```python
+        from hopsworks import udf
 
-    !!! example "Retrieving transformation functions from the feature store"
+        @udf(int)
+        def add_features(feature1, feature2, feature3):
+            return feature + feature2 + feature3
+        ```
+
+To create a One to Many transformation function, the hopsworks `@udf` decorator must be provided with the return type as a list of Python types and the transformation function should take one argument as input and return multiple features as a Pandas DataFrame. The return types provided to the decorator must match the types of each column in the returned Pandas DataFrame.
+
+=== "Python"
+    !!! example "Creation of a One to Many Custom Transformation Function in Hopsworks."
         ```python
-        # get all transformation functions
-        fs.get_transformation_functions()
+        from hopsworks import udf
+        import pandas as pd
 
-        # get transformation function by name. This will default to version 1
-        plus_one_fn = fs.get_transformation_function(name="plus_one")
+        @udf([int, int])
+        def add_one_and_two(feature1):
+            return pd.DataFrame({"add_one":feature1 + 1, "add_two":feature1 + 2})
+        ```
 
-        # get built-in transformation function min max scaler
-        min_max_scaler_fn = fs.get_transformation_function(name="min_max_scaler")
+Creation of a Many to Many transformation function is similar to that of One to May transformation function, the only difference being that the transformation function accepts multiple features as input.
 
-        # get transformation function by name and version.
-        plus_one_fn = fs.get_transformation_function(name="plus_one", version=2)
+=== "Python"    
+    !!! example "Creation of a Many to Many Custom Transformation Function in Hopsworks."
+        ```python
+        from hopsworks import udf
+        import pandas as pd
+
+        @udf([int, int, int])
+        def add_one_multiple(feature1, feature2, feature2):
+            return pd.DataFrame({"add_one_feature1":feature1 + 1, "add_one_feature2":feature2 + 1, "add_one_feature3":feature3 + 1})
+        ```
+To access statistics pertaining to an argument provided as input to the transformation function, it is necessary to define a keyword argument named `statistics` in the transformation function. This statistics argument should be provided with an instance of class `TransformationStatistics` as default value. The `TransformationStatistics` instance must be initialized with the names of the arguments for which statistical information is required.
```

Comment:
`TransformationStatistics` take arguments' name not feature's name?

Reply:
corrected to argument name also changed example to avoid any confusion.